codegen/kuberesource/crd.go

```diff
@@ -142,6 +164,16 @@ func CustomResourceDefinition(
 			},
 		},
 	}
+	if !options.WithoutSpecHash {
+		// hashstructure of the crd spec:
+		specHash, err := hashstructure.Hash(crd.Spec, nil)
```

Comment:
The CRD status can also cause processing errors—can we include the status in the hash as well? Either as a separate annotation or combined with the spec hash

Reply:
i believe the schema of the status is included in the crd Spec, so this should cover the status as well

Reply:
ah my bad—this is the spec of the _CRD definition itself_, not the spec of the CR